sql/schema.sql

```diff
@@ -6,31 +6,31 @@
 -----------------
 
 CREATE TABLE IF NOT EXISTS repos
-( id               INT  NOT NULL
-, repo_name        TEXT NOT NULL
-, repo_desc        TEXT NOT NULL
-, repo_url         TEXT NOT NULL -- The repo html_url
-, open_issue_count INT  NOT NULL
-, fork_count       INT  NOT NULL
+( id         SERIAL       
+, repo_name  TEXT   NOT NULL
+, repo_owner TEXT   NOT NULL
+, repo_desc  TEXT
+-- The repo's html_url
+, repo_url   TEXT   NOT NULL 
```

Comment:
I think we don't need to store `repo_url`. It can be constructed from owner and name

Reply:
Got it.